src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,83 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const StyledTabButton = styled(Button)`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabs = styled(LeafyTabs)`
+    justify-items: flex-end;
+    button:first-of-type {
+        margin-right: auto;
+    }
+
+    button[aria-selected='true'] {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const TabButton = ({ ...props }) => (
+    <StyledTabButton renderAsButton {...props} />
+);
+
+const Tabs = ({ tabsList, handleClick, activeItem }) => {
+    const [activeTabIndex, setActiveTabIndex] = useState(0);
+    const adjustTabIndex = activeTab => {
+        const index = tabsList.indexOf(activeTab);
+        if (index !== -1) {
+            setActiveTabIndex(index);
+        }
+    };
+
+    /*
+    to set the value from tab context in nav bar
+    */
+    useEffect(() => {
+        adjustTabIndex(activeItem);
+    }, [activeItem]);
+
+    /*
+    to set the active item upon user click on any leafy tabs
+    */
+    const onClick = useCallback(
+        index => {
+            const tabValue = tabsList[index];
+            adjustTabIndex(tabValue);
+            handleClick(tabValue);
+        },
+        [tabsList]
+    );
+
+    return (
+        <>
```

Comment:
Last nitpick from me, we shouldn't need the React fragments `<></>` because we are rendering one component here

Reply:
Good catch and thanks for the clarification.I had this question, but the other tabs.js had it, and I believe even that is a single component, so I just left it as is thinking may be it is our format. Will fix it.

Reply:
Oh good catch, I will look into it. That may just be an error. So long as we return one component then it is fine to omit.